migrations/sql/V2021.04.09.1500__SubnationalDemography.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE "subnational_region" (
+"id" SERIAL NOT NULL ,
+"country" INTEGER NOT NULL ,
+"name" TEXT NOT NULL ,
```

Comment:
how about a "level" field or something to indicate "province"/"state"/nhs_region or whatever

Reply:
Yeah, I wondered about this - it's not needed at the moment, as everything is at ADM1 level, and I worry that available data won't convincingly support anything better. I thought about an "admin_level" integer, and a nullable "parent_id" linking to the next level up, and then thought against... but it would be easy to put back.

Reply:
Added...

Reply:
I think anything is fine for now - I can imagine getting a second level of data for some country and that causing drama